src/api/fragments.js

```diff
@@ -60,3 +63,21 @@ export const deleteFragment = fragmentCode => (
     useAuthentication: true,
   })
 );
+
+export const getFragmentSettings = () => (
+  makeRequest({
+    uri: '/api/fragmentsSettings/',
```

Comment:
We are not adding a trailing slash at the end of the API URIs... could you remove it?

`uri: '/api/fragmentsSettings',`